recipe/meta.yaml

```diff
@@ -100,7 +100,7 @@ source:
 
 build:
   number: 0
-  skip: true  # [osx]
+  skip: true
```

Comment:
```suggestion
```

Reply:
oh man, it's too early to be fighting this beast, perhaps.